### PR TITLE
i18n(ja): Update contribute.mdx

### PR DESCRIPTION
**Pull request by morinokami**
#### Description (required)

- Update ja/contribute.mdx.
- [diff](https://github.com/withastro/docs/commits/main/src/content/docs/en/contribute.mdx?since=2023-07-24T17:28:42.000Z)
  - https://github.com/withastro/docs/pull/5014
  - https://github.com/withastro/docs/pull/5566

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/ja/contribute.mdx [hunk 1]
```diff
@@ -3,13 +3,12 @@ title: Astroに貢献する
...
 i18nReady: true
 ---
-import Button from '~/components/Button.astro'
 import ContributorList from '~/components/ContributorList.astro'
-import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
-
 
 私たちは、どんな規模の貢献も、どんなスキルレベルの貢献者（コントリビューター）も歓迎します。オープンソースのプロジェクトとして、コントリビューターに対し報いることが重要だと考えています。PR、テクニカルライティング、提案された機能の実現のために、喜んでガイダンスを提供します。
 
+より深く関わりたいですか？さまざまな役割、メンテナーの指名プロセス、コードレビュープロセス、行動規範の執行に関する詳細については、[ガバナンスドキュメント](https://github.com/withastro/.github/blob/main/GOVERNANCE.md)を参照してください。
```
Comment (morinokami) [inline src/content/docs/ja/contribute.mdx:10]: ここや「貢献の種類」の最初の段落など、diff に挙げていない部分も差分があったため、目視で差分確認し最新に合わせています。

File: src/content/docs/ja/contribute.mdx [hunk 2]
```diff
@@ -24,38 +23,30 @@ Astroプロジェクトに貢献する方法はたくさんあります！すべ
...
+初めてのオープンソースへの貢献であっても、あなたが作成した新しいAstroの機能に関するドキュメントを追加したい場合であっても、経験豊富な翻訳者であるあなたが次に翻訳するページを探している場合であっても、PRレビュアーとしての貢献について学びたい場合であっても、私たちはあなたをサポートします！
 
-- [メンテナガイド](https://github.com/withastro/docs/blob/main/contributor-guides/astro-maintainers-guide-to-docs.md) - Astroメンテナーとしてこのリポジトリをメンテナンスする方法。
-- [サポート部隊ガイド](https://github.com/withastro/docs/blob/main/contributor-guides/support-squad-guide-to-docs.md) - サポート担当としてTeam Docsをサポートする方法。
+誤字修正者、ライター、翻訳者、機能の作成者、さらにはPRレビュアーとしてあなたがAstro Docsへ貢献するために作成されたドキュメントを、専用サイト[*Astro Docs* Docs](https://contribute.docs.astro.build)にアクセスして確認してください。
 
 ## コントリビューター
 
-このドキュメントは、親切な人たちによってもたらされたものです。[GitHubで参加しましょう！](https://github.com/withastro/docs)
+このドキュメントは、多くの方々の協力によって提供されています。[GitHubで参加しましょう！](https://github.com/withastro/docs)
```
Comment (morinokami) [inline src/content/docs/ja/contribute.mdx:50]: ちょっとおかしな訳だったため、https://github.com/withastro/docs/pull/3550 の修正を revert しています（そもそも revert を提案していたのですが、それがマージされ忘れていたようです）。

**Review by okonomi — APPROVED** [review]
LGTMです！

**Comment by sarah11918** [comment]
Hi! Just a note that there was a recent update to contributing.mdx that no longer lists out the individual guides for contributing to docs. If you check to the current English for "Contribute to the docs" it should look like that, with just a single link to Astro Docs Docs.

That should help you resolve the conflict... replace it entirely! :smile:

**Comment by morinokami** [comment]
@sarah11918 Resolved the conflict, thanks!

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**